storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -0,0 +1,32 @@
+[id="persistent-storage-csi-ebs"]
```

Comment:
Is this included in the topic map? I can't see it in the preview to check the links.

Reply:
@bmcelvee Yes, it is included in the topic map. 
```
- Name: AWS Elastic Block Store CSI driver Operator
  File: persistent-storage-csi-ebs
```
Links seem to be working from the preview, such as https://osdocs-963--ocpdocs.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-ebs.html#persistent-storage-csi-ebs

Or do you mean something else?